src/analysis/calculus/times_cont_diff.lean

```diff
@@ -1380,6 +1380,28 @@ by simp [times_cont_diff_on_univ.symm, differentiable_on_univ.symm, fderiv_withi
          - fderiv_within_univ, times_cont_diff_on_succ_iff_fderiv_within unique_diff_on_univ,
          -with_bot.coe_add, -add_comm]
 
+/-- A function is `C^(n + 1)` at a point iff locally, it has a derivative which is `C^n`. -/
+theorem times_cont_diff_at_succ_iff_has_fderiv_at {n : ℕ} :
```

Comment:
Could you move this lemma to the part on smooth functions at a point, for instance around line 1208?